File: alpaul_gls/Content/Downloadable/PHAO2018092520181018161350/Business/AspNetUserLogin.vb
Imports System.Collections.Generic

Public Class AspNetUserLogin
    Public Shared Function Add(value As Models.AspNetUserLogin) As Integer
        Return Data.AspNetUserLogin.Add(value)
    End Function

    Public Shared Function Edit(value As Models.AspNetUserLogin) As Integer
        Return Data.AspNetUserLogin.Edit(value)
    End Function

    Public Shared Function Delete(value As String) As Integer
        Return Data.AspNetUserLogin.Delete(value)
    End Function

    Public Shared Function GetById(value As String) As Models.AspNetUserLogin
        Return Data.AspNetUserLogin.GetById(value)
    End Function

    Public Shared Function GetByGUID(value As String) As Models.AspNetUserLogin
        Return Data.AspNetUserLogin.GetByGUID(value)
    End Function

    Public Shared Function GetAll() As List(Of Models.AspNetUserLogin)
        Return Data.AspNetUserLogin.GetAll()
    End Function
End Class

File: alpaul_gls/Content/Downloadable/PHAO2018092520181018171704/Business/AspNetUserLogin.cs
using System.Collections.Generic;

namespace Business
{
    public class AspNetUserLogin
    {
	    private static Data.AspNetUserLogin obj = new Data.AspNetUserLogin();

        public static int Add(Models.AspNetUserLogin value)
        {
            return obj.Add(value);
        }

        public static int Edit(Models.AspNetUserLogin value)
        {
            return obj.Edit(value);
        }

        public static int Delete(stirng value)
        {
            return obj.Delete(value);
        }

        public static Models.AspNetUserLogin GetById(string ID)
        {
            return obj.GetById(ID);
        }

		public static Models.AspNetUserLogin GetByGUID(string ID)
        {
            return obj.GetByGUID(ID);
        }

        public static List<Models.AspNetUserLogin> GetAll()
        {
            return obj.GetAll();
        }
    }
}